.github/workflows/main.yml

```
@@ -0,0 +1,145 @@
+
+          PR_BODY=$(cat "$PART" | sed 's/"/\\"/g' | jq -Rs .)
+
+          JSON_STRING=$(jq -n \
+            --arg title "$PR_TITLE" \
+            --arg body "$PR_BODY" \
+            '{
+              messages: [
+                {
+                  role: "system",
+                  content: "あなたは優秀なテックリードです。最新のC#,Unityの基本的なコードの書き方は熟知しており、それらのコンポーネントやUnityのゲームのアンチパターンなどに関しても世界でも有数の知見を持っています。\n
+以下のルールに従ってコードレビューを行ってください。\n
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 83: could not find expected ':'

(syntax-check)